app-office/whid/whid-2.0.2~beta.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="Time Tracker for Freelancers and Individual Contractors"
+DESCRIPTION="What Have I Done - Time Tracker for Freelancers and Individual Contractors. \
+Features: \
+* Very easy and intuitive to use  \
+* Measures time for active tasks \
+* Maintains a tree-structures of folders, customers, projects and tasks \
+* Reports time spent, breakdowns for each day in a week \
+* Reports time spent on selected items in the tree structure of tasks \
+* Stores the data in SQLite \
+* Based on 20 years of experience and two previous iterations designing and implementing time \
+  tracking applications \
+* It's free, contains no ads, does not track you, and stores your data on your local computer. \
```

Comment:
In DESCRIPTION every line should be wrapped at 80 column.

Reply:
ok done